core/src/main/java/org/opensearch/sql/calcite/udf/textUDF/replaceFunction.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.calcite.udf.textUDF;
+
+import org.opensearch.sql.calcite.udf.UserDefinedFunction;
+
+/**
+ * We don't use calcite built in replace since it uses replace instead of replaceAll
+ */
+
+public class replaceFunction implements UserDefinedFunction {
+    @Override
+    public Object eval(Object... args) {
+        if (args.length != 3) {
+            throw new IllegalArgumentException("replace Function requires 3 arguments, but current get: " + args.length);
+        }
```

Comment:
Can we refactor this to a general util method? Seems this similar code block is duplicated in almost Function classes.

Reply:
Can we firstly remain it here? I'm considering reuse the check argument function in

Reply:
Sure, please fix it in followup PR.